federationapi/routing/routing.go

```diff
@@ -131,7 +137,7 @@ func Setup(
 				cfg, rsAPI, keyAPI, keys, federation, mu, servers, producer,
 			)
 		},
-	)).Methods(http.MethodPut, http.MethodOptions)
+	)).Methods(http.MethodPut, http.MethodOptions).Name(SendRouteName)
```

Comment:
It's useful for route unit testing. 
It registers the route using the provided name with the mux so it can be retrieved in a test and passed to `httptest`.
ie. `federationapi/routing/send_test.go`

Reply:
Not necessary now that I refactored the tests.